### PR TITLE
[Vue] add intermediate proxy script for Vue CLI bin that fixes typescript error lines

### PR DESCRIPTION
**Pull request by diosmosis**
### Description:

When processing .vue files, Vue CLI bin will find the `<script lang="ts">` element, and feed its contents into the typescript compiler. The line numbers for errors are specific to that chunk of text and not the entire .vue file. This PR adds an intermediate script that corrects the line numbers in those errors.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/CoreVue/Commands/Build.php [hunk 1]
```diff
@@ -123,13 +123,14 @@ private function watch($plugins, $printBuildCommand, OutputInterface $output)
             $output->writeln("<comment>$command</comment>");
             return;
         }
-        passthru($command);
+
+        passthru($commandSingle);
```
Comment (sgiehl) [inline plugins/CoreVue/Commands/Build.php]: Was that change intended? I didn't test it, but looks like it would try to execute the command containing placeholders now.

Comment (diosmosis) [reply plugins/CoreVue/Commands/Build.php]: no not intentional :+1:

**Review by peterhashair — APPROVED** [review]
Looks good to me
